cmd/swarmctl/service/list.go

```diff
@@ -31,6 +31,17 @@ var (
 				return err
 			}
 
+			nr, err := c.ListNodes(common.Context(cmd), &api.ListNodesRequest{})
+			if err != nil {
+				return err
+			}
+			readyNodes := make(map[string]struct{})
+			for _, n := range nr.Nodes {
+				if n.Status.State == api.NodeStatus_READY {
```

Comment:
I think it might be better to change the condition to `n.Status.State != api.NodeStatus_DOWN`. Other node states are live or transient. For example, a node reconnecting to a new manager, i.e., NodeStatus_DISCONNECTED, we don't want to show these tasks as down.